pkg/process/exec_conf.go

```diff
@@ -217,11 +232,11 @@ func cleanup(cmd *cobra.Command) {
 		// okay now that logging is working, inform about the broken keys
 		if cmd.Annotations["type"] != "helper" {
 			for _, key := range brokenKeys {
-				logger.Sugar().Infof("Invalid configuration file key: %s", key)
+				logger.Sugar().Fatalf("Invalid configuration file key: %s", key)
```

Comment:
Does this mean that environment variables will cause a warning "Invalid configuration _file_ key" ?

Reply:
It will stop the program from starting completely.

Reply:
I'll be more blunt. Can this be changed to include "environment variable" in the error message about environment variables please?

Reply:
that will require more work but okay

Reply:
whoops, actually, it turns out it will be painfully obvious already. the only way this line happens is if an env var that starts with STORJ_ isn't expected. which means that when it's a config file mistake it will look like "debug.missing_key" and when it's an env var it will look like "STORJ_DEBUG_MISSING_KEY" but i'll take out the explicit mention of a file. sufficient?

Reply:
I think that's ok.